vendor/calcite-1_20_0/build.gradle.kts

```diff
@@ -16,18 +16,21 @@
  * limitations under the License.
  */
 
-plugins { id 'org.apache.beam.vendor-java' }
+plugins {
+    id("org.apache.beam.dependencies")
```

Comment:
I'm confused about how this worked originally. Shouldn't this have needed to depend on beamModule in order to access `library.java`?

Reply:
I'm confused too. I think it is because of groovy's dynamic nature